lib/core/src/Cardano/Pool/DB/Sqlite/TH.hs

```diff
@@ -76,4 +76,13 @@ PoolOwner sql=pool_owner
 
     Primary poolOwnerPoolId poolOwnerOwner
     deriving Show Generic
+
+-- Mapping of metadata to pool
+PoolMetadata sql=pool_metadata
```

Comment:
**[remark]**

good idea to decouple PoolMetadata from PoolOwner. The former one is expected to have different update rate. And here I am thinking once again about repsert comment from above :thinking:

Reply:
They are actually updated at the very same time (when a registration certificate is discover on-chain). The reason they are decoupled is because there's a many:many relationship between pools and owners (one pool can have multiple owners, and an owner multiple pools).